### PR TITLE
fix a todo: wrap some configuration into EtcdConf

### PR DESCRIPTION
**Pull request by zzl200012**
### What problem does this PR solve?

Issue Number: #329  

Problem Summary:
simple wrapping

### What is changed and how it works?

What's Changed:

How it Works:

Side effects(Breaking backward compatibility? Performance regression?): 

### Check List

- [x] Relevant documentation/comments is changed or added
- [x] I acknowledge that all my contributions will be made under the project's license

### CODE REVIEW

**Comment by ilixiaocui** [comment]
recheck

File: include/etcdclient/etcdclient.h [hunk 1]
```diff
@@ -88,6 +88,8 @@ struct EtcdConf {
     char *Endpoints;
     int len;
     int DialTimeout;
+    int etcdTimeout;
```
Comment (ilixiaocui) [inline include/etcdclient/etcdclient.h:91]: Unit test compilation failed
http://59.111.91.248:8080/job/curve_untest_job/982/console

**Comment by aspirer** [comment]
recheck

**Comment by wu-hanqing** [comment]
recheck

**Comment by wu-hanqing** [comment]
@zzl200012 cpplint test failed, and unittest compilation failed,  http://59.111.91.248:8080/job/opencurve_multijob/1415/
image

**Comment by ilixiaocui** [comment]
cc @zzl200012 Please resubmit to the master branch if necessary.

**Pull request closed without merging**